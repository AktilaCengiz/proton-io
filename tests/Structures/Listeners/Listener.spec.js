const Listener = require("../../../src/Structures/Listeners/Listener");
const checkEqualKeys = require("../../checkEqualKeys");

test("done", () => {
    const _ = new Listener("_", {
        emitter: "client",
        event: "ready",
        type: "once"
    });

    checkEqualKeys(_, {
        emitter: "client",
        event: "ready",
        type: "once",
        category: "default",
        client: null,
        handler: null,
        filepath: null
    });

    expect(_).toMatchSnapshot();
});
