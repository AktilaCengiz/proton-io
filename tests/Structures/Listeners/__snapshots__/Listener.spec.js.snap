// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`properties 1`] = `
Listener {
  "category": "default",
  "client": null,
  "emitter": "client",
  "event": "ready",
  "filepath": null,
  "handler": null,
  "id": "_",
  "type": "once",
}
`;
